components/director/internal/domain/formationconstraint/entity.go

```diff
@@ -0,0 +1,22 @@
+package formationconstraint
```

Comment:
Add in the new components folders an OWNERS file similar to the other that we have + adjust the CODEOWNERS file as well - example `/components/director/internal/formationmapping @kyma-incubator/team-raptor` but replace the folder